packages/react/src/internal/internal-feature-app-container.tsx

```diff
@@ -24,11 +25,7 @@ export const handleError = (
   }
 };
 
-export interface BaseFeatureAppContainerProps<
-  TFeatureApp,
-  TFeatureServices extends FeatureServices = FeatureServices,
-  TConfig = unknown
-> {
+export interface FeatureAppDescriptor {
```

Comment:
Maybe import the interface from `packages/core/src/feature-app-manager.ts`

Reply:
I didn't do that because it has (intentionally) different jsdoc comments for the properties.